src/components/ProductTile.vue

```diff
@@ -18,9 +20,27 @@ const { product } = toRefs(props);
         <div class="relative mx-3 flex h-max-[275px] overflow-hidden justify-center">
             <ProductImage :product="product"/>
             <div class="absolute top-0 left-0 flex gap-1">
-                <span v-if="isPromotion" class="rounded bg-brand-200 px-2 py-0.5 text-center text-xs font-medium text-white">
+                <span v-if="isPromotion"
+                      class="rounded bg-brand-200 px-2 py-0.5 text-center text-xs font-medium text-white">
                     SPONSORED
                 </span>
+                <Popover v-if="isPromotion" placement="bottom-start">
```

Comment:
Jeg ville nok have puttet denne ind i den blå promotion boks for bedre at vise at de hører sammen :)